databases/week2/part_two.sql

```diff
@@ -0,0 +1,57 @@
+DROP DATABASE IF EXISTS `hw_lesson2_part2`;
+
+CREATE DATABASE IF NOT EXISTS `hw_lesson2_part2`;
+
+USE `hw_lesson2_part2`; 
+
+CREATE TABLE class (
+	id INT(10)NOT NULL AUTO_INCREMENT PRIMARY KEY, 
+    class_name VARCHAR(255) NOT NULL, 
+    class_begins DATE NOT NULL, 
+    class_ends DATE NOT NULL 
+); 
+
+CREATE TABLE student (
+	id INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY, 
+    name VARCHAR(255) NOT NULL, 
+    email VARCHAR(255) UNIQUE NOT NULL, 
+    phone VARCHAR(255) NULL,
```

Comment:
Please note that it is not ideal to assign a human name, email, or phone with 255 characters in length. Please adjust as below for instance
```
name VARCHAR(20) NOT NULL, 
    email VARCHAR(20) UNIQUE NOT NULL, 
    phone VARCHAR(15) NULL,
```